.github/workflows/generate-site.yaml

```
@@ -46,7 +46,7 @@ jobs:
       success: ${{ steps.build-site.outputs.success }}
       is-bot: ${{ steps.is-bot.outputs.bot }}
     container:
-      image: ghcr.io/hubverse-org/hub-dash-site-builder:latest
+      image: ghcr.io/hubverse-org/hub-dash-site-builder:znk-dispatch-fix-34
```

This will change back to latest when we create a release.

---

to test the most recent build

```suggestion
      image: ghcr.io/hubverse-org/hub-dash-site-builder:main
```